packages/curvenote/src/works/create.ts

```diff
@@ -0,0 +1,55 @@
+import { selectors } from 'myst-cli';
+import type { ISession } from '../session/types.js';
+import { confirmOrExit, uploadContentAndDeployToPrivateCdn } from '../utils/index.js';
+import chalk from 'chalk';
+import { postNewWork, postNewWorkVersion } from './utils.js';
+import { loadTransferFile, upwriteTransferFile } from './transfer.js';
+
+export async function create(session: ISession, opts?: { ci?: boolean; yes?: boolean }) {
+  if (session.isAnon) {
+    throw new Error(
+      '⚠️ You must be authenticated for this call. Use `curvenote token set [token]`',
+    );
+  }
+
+  const siteConfig = selectors.selectCurrentSiteConfig(session.store.getState());
+  if (!siteConfig) {
+    throw new Error('🧐 No site config found.');
+  }
+
+  const transferData = await loadTransferFile();
```

Comment:
@fwkoch can you point me to a better way to be interacting with this file. Right now i'm reading/writing in the current folder but really/probably this file should be created and maintained adjacent to the root curvenote/myst.yml file?

I'm not sure how to achieve that, and double unsure of how to do that without involving any changes to mystmd.